tools/amd/build.js

```diff
@@ -37,7 +38,7 @@ export default function BuildBower() {
     .then(() => Promise.all([
       bowerConfig(),
       generateFactories(factoriesDestination, babelOptions),
-      exec(`babel ${babelOptions} ${srcRoot} --out-dir ${libDestination}`),
+      buildFolder(srcRoot, libDestination, babelOptions),
```

Comment:
The biggest problem with the cli is that there is no way (that I know) to run it from stdin in a cross platform way.
Therefore, I thought using the api instead would be cleaner. Since I changed to use the api for factories, it would be more uniform to use it everywhere.

That said, I think the solution to have write a temporary file then run babel and delete the temp file is also valid. I wanted to show another option to see which we prefer.